docs/docs/get-started/basic-concepts.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Basic Concepts
+sidebar_label: Basic Concepts
+slug: '/basic-concepts'
+sidebar_position: 4
+---
+
+Package
```

Comment:
I would add a sentence here to take readers to the "Advanced Concepts" section for Packages if they want to learn more.